assignment5.text

```diff
@@ -0,0 +1,11 @@
+1. call it by typing fxn(a, b, c) where a, b, and c would be constants or variables inputted by the user. It could look like this...
+
+void main() {
+	printf("Enter a number: \n");
+	scanf("%d", a);
+... and then two more lines with b being a float and c being an int.
+	}
```

Comment:
this is right, but you're missing the ```int fxn(int a, float b, int b);``` that's needed in the beginning, before the main. Without it, you'll get a warning!